cheatsheets_draft/Legacy_Application_Management_Cheat_Sheet.md

```diff
@@ -0,0 +1,75 @@
+
+## Ensuring Maintainability
+
+Where possible maintain a high degree of institutional expertise regarding the application, so that it is possible for staff to both remediate security vulnerabilities and troubleshoot the application when needed to ensure business continuity. The following recommendations apply:
+
+- More than one staff member should be adequately trained to troubleshoot/reconfigure the legacy application as needed. This will reduce the risk of complete loss of maintenance capability for the legacy application if one trained member of staff leaves the organisation.
+- Encourage staff to regularly document processes including recording troubleshoot guides for common failure scenarios for the legacy application.
+- It might be necessary to teach a core group of staff to write basic programs in the language used by the legacy application, where this expertise does not exist in your organisation.
+
+## Change Management
+
+The ultimate goal for most legacy applications will be to migrate, when possible, from unmaintainable to a solution which is both maintainable and architected to be resilient to contemporary threats. Staged change management can take into account the following factors:
+
+- What budget can practically be allocated to upgrading to a modern solution and within what time frame could budget be made available?
+- Do people with the necessary expertise required to handle migration exist within your organisation or could these people be acquired/developed?
+- How urgently does a migration to an upgraded solution need to happen based on the risk profile of the application and the risk appetite of your organisation?
+
+A change management plan, formal or informal, should include a clear description of granular steps towards migration to an upgraded solution, an explicit date of expected completion, and a clear articulation of the business and security case for the change. To produce a realistic plan for migration, staff involved in overseeing and using the existing solution should be consulted extensively to get a sense for how critical the legacy application is to your organisation and what barriers there might be to facilitating migration.
+
+## Continuous Monitoring and Incident Response
+
+Legacy applications should be subject to an especially high degree of security monitoring with rapid response efforts made to investigate potential incidents. This might be challenged by intra-operability issues that mean that logs produced by the application are in a format that cannot be ingested by security monitoring tools used by your organisation. Potential work arounds might include:
```

Comment:
```suggestion
Legacy applications should be subject to an especially high degree of security monitoring with rapid response efforts made to investigate potential incidents. This might be challenged by intra-operability issues that mean that logs produced by the application are in a format that cannot be ingested by security monitoring tools used by your organization. Potential workarounds might include:
```

Reply:
And changed.